src/BinSkim.Rules/PERules/WindowsBinaryAndPdbSkimmerBase.cs

```diff
@@ -130,20 +134,27 @@ public static void LogExceptionLoadingPdb(IAnalysisContext context, PdbException
                 throw new ArgumentNullException(nameof(context));
             }
 
-            // '{0}' was not evaluated for check '{1}' because its PDB could not be loaded ({2}).
+            string fileName = context.TargetUri.GetFileName();
```

Comment:
>fileName [](http://example.com/codeflow?start=19&length=8)

key sould also consider path?
/x86/a.exe
/x64/a.exe

Reply:
Changed to OriginalString as path. That will prevent issues like you showed.

thanks!